unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NotificationMessagesHUD/NotificationComponent/IChatNotificationMessageComponentView.cs

```diff
@@ -40,6 +40,12 @@ public interface IChatNotificationMessageComponentView
     /// <param name="isPrivate">If the notification is private or not.</param>
     void SetIsPrivate(bool isPrivate);
 
+    /// <summary>
+    /// Set the notification type, can be a friend request or not.
+    /// </summary>
+    /// <param name="isAccepted">If the notification is private or not.</param>
+    void SetIsFriendRequest(bool isAccepted);
```

Comment:
feels kind of odd to consider a `IChatNotificationMEssageComponentView` to a different "friend request type"